### ISSUE TITLE
Add option to remove Exif viewer

### ISSUE THREAD

**Issue by matidio** [odebroqueville/contextSearch#20]
Please delete the exif viewer in this extension. Until now the extension was only about search engines and this was good so. One extension for one scope.
I don't know what an exif viewer has to do with a search engine extension.
If you like to have your own exif viewer in a browser why don't you create another, separate extension with all its _related_ functionalities.
Til now we all were enthusiastic about your extension. I don't know whether we'll continue to be so in future if you integrate various functionalities in one extension.

**Comment by odebroqueville** [comment]
Tough luck! Go use another extension if you're not happy.

**Comment by odebroqueville** [comment]
Context Search Lite has been released. It has the same functionality as Context Search but without the Exif Viewer. https://addons.mozilla.org/fr/firefox/addon/context-search-lite/

**Comment by rnkn** [comment]
Hello,
I came here to ask about this too. Thanks for creating the extension and the lite version. The inclusion of the Exif viewer is a bit confusing. Maybe your efforts could be even better appreciated if you spun off the Exif viewer into its own extension?

**Comment by odebroqueville** [comment]
Hello Paul, thanks for using Context Search. Unfortunately, I no longer have enough time to further develop any extensions. It's been a very unrewarding experience! If you don't need the Exif Viewer then please just use the lite version. Cheers.

**Comment by rnkn** [comment]
I'm sorry to hear that the experience has been unrewarding for you. I hope you take some solace in knowing that your work continues to bring others like myself some happiness :)

**Comment by odebroqueville** [comment]
I'm obviously very pleased that so many users, like yourself, enjoy using the extension. But, being unemployed, I was hoping to make a bit of reasonable pocket money. Since its first release, Context Search, which counts thousands of lines of code and hundreds of hours of work, has received donations of a total of $30 by 3 generous donors. That's a bit disappointing, given the extension counts roughly 10'000 users. Anyway, glad some users have enjoyed the ride!

**Comment by rnkn** [comment]
Although I've not had the intention of making money on any of my open source projects, I've found the real advantage in creating projects is in making contacts. These contacts have then created paid opportunities for me that far outstripped anything I could bring in through donations. I think it is not just the code that impresses, but also showing that you have the vision to create something new and the skills to manage a project.

**Comment by odebroqueville** [comment]
Very interesting feedback! Maybe I haven't persevered enough! I was somewhat discouraged by the extreme difficulties I had to get help to debug my JS code. I tried to apply the latest developments in JS related to async code, and that's where I started encountering problems that I couldn't fix. To come back to your comment about Exif Viewer, Context Search is my first ever extension for Firefox and I did have the intention to keep adding features, which today I understand isn't the right way to go if the features are not context-search related. Today, I better understand that a modular approach is preferable where unrelated features are placed in separate extensions. It's part of the learning process! Thanks again for your interesting views, Paul. All the best to you.
